README.md

```diff
@@ -1007,6 +1008,25 @@ The initialization function is run from the context of the HTML element.
 It should be noted that children nodes are removed from the DOM when widget instantiation occurs, and added as children
 to the widget instance.
 
+#### Meta
```

Comment:
Put in a quick example of meta. I have examples for Dimensions and Intersection ready to go into their own PRs. What kind of info do you think you we need here?